test-common/BUILD.bazel

```diff
@@ -44,47 +44,41 @@ alias(
         ("model-tests", ".dar"),
         ("dar-files", ""),
         ("dar-files", "-lib"),
-        ("test-common", ""),
     ]
     for (name, lf_version) in lf_version_configuration.items()
 ]
 
-alias(
+da_scala_library(
     name = "test-common",
-    actual = "test-common-default",
-    visibility = ["//visibility:public"],
+    srcs = glob(["src/main/scala/**/*.scala"]),
+    scala_deps = [
+        "@maven//:org_scalatest_scalatest_shouldmatchers",
+        "@maven//:org_scalatest_scalatest_core",
+        "@maven//:org_scalactic_scalactic",
+        "@maven//:org_scalatest_scalatest_matchers_core",
+        "@maven//:org_scalaz_scalaz_core",
+    ],
+    tags = ["maven_coordinates=com.daml:test-common:__VERSION__"],
+    visibility = [
+        "//visibility:public",
+    ],
+    deps = [
+        "//daml-lf/archive:daml_lf_archive_reader",
+        "//daml-lf/data",
+        "//ledger-api/grpc-definitions:ledger_api_proto_scala",
+        "//ledger-api/testing-utils",
+        "//ledger/ledger-api-common",
+        "//ledger/ledger-api-domain",
```

Comment:
is this not annoying, if you want to use test-common in canton ? 
@mziolekda @nmarton-da @atriantafyllos-da

Reply:
Yes keeping the duplication of the file seems more suitable.

Reply:
that is annoying indeed but it should work. We don't rely on anything inside /ledger anymore, all the libraries therein have been cleaned up and moved to other directories.
This dependency will be a new "dirty dependency" of the test-common that no-one will need.

Another suggestion, could TestCommands be moved to ledger-api-client instead?